src/boss/bo4/unk_45354.c

```diff
@@ -109,7 +109,40 @@ void func_8010E570(s32 arg0) {
     SetPlayerAnim(D_us_80181320[anim]);
 }
 
-INCLUDE_ASM("boss/bo4/nonmatchings/unk_45354", func_8010E6AC);
+void func_8010E6AC(bool forceAnim13) {
+    bool atLedge;
+
+    atLedge = 0;
+    if (g_Dop.vram_flag & 0x20) {
+        atLedge = 1;
```

Comment:
Nitpick but since this is already defined as a bool might as well make these true/false